pages/page.py

```diff
@@ -56,13 +36,13 @@ def get_url_current_page(self):
     def is_element_present(self, *locator):
         self.selenium.implicitly_wait(0)
         try:
-            self._selenium_root.find_element(*locator)
+            self.selenium.find_element(*locator)
             return True
         except NoSuchElementException:
             return False
         finally:
             # set back to where you once belonged
-            self.selenium.implicitly_wait(self.testsetup.default_implicit_wait)
+            self.selenium.implicitly_wait(10)
```

Comment:
This isn't a change in value, it's just pytest-selenium doesn't set a default implicit wait.